docs/platforms/javascript/guides/react/features/react-router.mdx

```diff
@@ -23,15 +23,15 @@ _(Available in version 7 and above)_
 
 To use React Router v6 with Sentry:
 
-Add `Sentry.reactRouterV6BrowserTracingIntegration` instead of the regular `Sentry.browserTracingIntegration` and provide the functions it needs to enable performance tracing:
+Add `Sentry.reactRouterV6BrowserTracingIntegration` instead of the regular `Sentry.browserTracingIntegration` and provide the required react functions:
```

Comment:
```suggestion
Add `Sentry.reactRouterV6BrowserTracingIntegration` instead of the regular `Sentry.browserTracingIntegration` and provide the required React hooks and router functions:
```

L: Everything with a `use` is called a hook in React, so I don't know if people find it weird if we just call it "function" (technically it is one though).

Reply:
sounds good, thx!